### PR TITLE
Morelike: update recommend endpoint

### PR DESCRIPTION
**Pull request by kodchi**
Add a new return field, and change the existing field to match the
source.

Bug: T219505

### CODE REVIEW

File: v1/recommend.yaml [hunk 1]
```diff
@@ -154,7 +154,10 @@ components:
...
             description: Wikidata ID for the item
-          score:
+          normalized_rank:
             type: number
-            description: Score of the recommendation. The higher the score, the more
-              important the recommendation is.
+            description: Normalized rank of the recommendation. The
+              higher the rank, the more important the recommendation is.
+          source_language:
+            type:string
```
Comment (Pchelolo) [inline v1/recommend.yaml]: You need a space after `:` or it's not valid yaml.

**Comment by Pchelolo** [comment]
It would be neat to add a schema test for recommendation API. See https://github.com/wikimedia/restbase/blob/master/test/features/schema_tests.js for inspiration

**Comment by berndsi** [comment]
Changes like this should probably bump the version number.

**Comment by kodchi** [comment]
> Changes like this should probably bump the version number.

Do you suggest a major version bump (because it's kind of a breaking change)? Since the API is marked as alpha quality, I was holding off on bumping the version as it may shoot up fast until we reach beta or stable, aka Chrome versioning.

**Comment by berndsi** [comment]
Well, I commented before I looked up the version number. What I meant was the version number exposed in the response content-type header. But it looks like it's not set for this yet. (Compare the string after `profile` in https://en.wikipedia.org/api/rest_v1/#/Mobile/get_page_mobile_sections__title_.)

I guess you can ignore this for now until you add version numbers to your responses.

**Comment by kodchi** [comment]
Some of the tests are failing because of https://phabricator.wikimedia.org/T215222. Can we still merge the pull request while we work to fix the above bug?

**Comment by Pchelolo** [comment]
> Can we still merge the pull request while we work to fix the above bug?

No, this would not be good. It will make RESTBase test suite fail for other patches and having unrelated test failures makes it much less convenient to work.

We can merge it as soon as you deploy the updated fixed version of the recommendation-api.

**Comment by kodchi** [comment]
OK, that makes sense.

**Comment by d00rman** [comment]
@kodchi needs a rebase at least.

**Comment by d00rman** [comment]
Closing due to inactivity

**Pull request closed without merging**